docs/Versioning.md

```diff
@@ -53,13 +66,33 @@ As a general principle, implementations SHOULD be robust in the face of missing
 
 By way of example, the `ModelProto.ir_version` MUST be present in every model.  The ONNX checker (`onnx/checker.py`) will enforce these rules.
 
-ISSUE: decide and document how we want to handle changes to the format that are forward compatible but will cause the generated code to introduce a breaking change (e.g., changing the data type of a field from int64 to int32, int64 to double or bytes to string).  I would recommend that we never do it after we hit 1.0.0 - prior to that we should be fairly liberal with changes to size (e.g., `int32<>int16`, `float<>double`) and conservative with changes to value space (e.g., `integral<>floatingpoint`, `string<>int`, `scalar<>message`)
+Because onnx.proto is expected to be consumed by multiple independent developers, changes to onnx.oroto SHOULD NOT break code that depends on generated language bindings (e.g., changing the type of an existing field).
```

Comment:
Confirm my understanding please. So a type change of an existing field will introduce a IR major version change, am I right? (although it's data compatible in some cases). Thank you!

Reply:
This is a good question. I think as written here, it would be a major version change. However, I am not convinced this is necessarily what we should do; I'd rather revisions to major versions indicating something like, "Hey! Consumer! You need to do something different!" Which wouldn't be the case here.